planning/autoware_path_optimizer/include/autoware/path_optimizer/mpt_optimizer.hpp

```diff
@@ -110,7 +183,7 @@ class MPTOptimizer
     const TrajectoryParam & traj_param, const std::shared_ptr<DebugData> debug_data_ptr,
     const std::shared_ptr<autoware::universe_utils::TimeKeeper> time_keeper_);
 
-  std::vector<TrajectoryPoint> optimizeTrajectory(const PlannerData & planner_data);
+  std::pair<std::vector<TrajectoryPoint>, int> optimizeTrajectory(const PlannerData & planner_data);
```

Comment:
Instead of using std::pair with int, please use std::optional<std::vector<TrajectoryPoint>>.